routes/members.js

```diff
@@ -1,6 +1,8 @@
 const express = require('express')
 const router = express.Router()
 const members = require('../controllers/members')
+const recruitersController = require('../controllers/recruiters')
+const recruiterValidator = require('../middlewares/validators/recruiter')
```

Comment:
NIT: If I was you, I would have destructured these two as well.

```javascript
const { addRecruiter } = require('../controllers/recruiters')
const { validateRecruiter } = require('../middlewares/validators/recruiter')
```
And use this value on line no 85

Reply:
I have destructured above as you suggested.